twine-upload.sh

```diff
@@ -120,7 +120,7 @@ if [[ ${INPUT_VERIFY_METADATA,,} != "false" ]] ; then
     twine check ${INPUT_PACKAGES_DIR%%/}/*
 fi
 
-TWINE_EXTRA_ARGS=
+TWINE_EXTRA_ARGS="--disable-progress-bar"
 if [[ ${INPUT_SKIP_EXISTING,,} != "false" ]] ; then
     TWINE_EXTRA_ARGS=--skip-existing
```

Comment:
Don't forget that this overrides the variable. It now has to append to it.

Reply:
Oh, that's not going to work :(
That line is the initial setting of the var and this one will overwrite it so appending should happen here.